src/parser/jobs/ast/modules/Draw.tsx

```diff
@@ -162,10 +174,10 @@ export default class Draw extends Module {
 		// Prepull consideration: + 1 play
 
 		// Begin Theoretical Max Plays calc
-		const fightDuration = this.parser.fight.end_time - this.parser.fight.start_time
-		const maxSleeveUses = Math.floor((fightDuration - (CARD_DURATION*2)) / (this.data.actions.SLEEVE_DRAW.cooldown * 1000)) + 1
+		const fightDuration = this.parser.pull.duration
+		const maxSleeveUses = Math.floor(Math.max(0, (fightDuration - (CARD_DURATION*2))) / (this.data.actions.SLEEVE_DRAW.cooldown * 1000)) + 1
 		const playsFromSleeveDraw = maxSleeveUses * SLEEVE_DRAW_PLAYS_GIVEN
-		const playsFromDraw = Math.floor((fightDuration - CARD_DURATION) / (this.data.actions.DRAW.cooldown * 1000)) + 1
+		const playsFromDraw = Math.floor(Math.max(0, (fightDuration - CARD_DURATION)) / (this.data.actions.DRAW.cooldown * 1000)) + 1
 		const theoreticalMaxPlays = playsFromDraw + playsFromSleeveDraw + 1
```

Comment:
That's something new. In 5.2 we didn't need to prepull draw, and the general consensus was that it's not important to hold your group back 30 seconds just to have it up. So we tried not to penalize on it. But in 5.3, it's a HUGE dps loss not to have div up when it's supposed to. So yea let's do that. I'll add that over the weekend

Reply:
What do you think about removing prepull draws from the card play checklist if the fight is from pre-5.3? Since you said it wasn't considered important in 5.2, it was probably relatively uncommon, so including it would artificially lower the checklist score/percentage in historical fights compared to 5.3.
(I swear I'm not trying to make a bunch of extra work for you - I'd be willing to make that update after you finish this one if you think it's a good idea)